abipy/lessons/lesson_helper_functions.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+"""
+reusable functions for lessons
+"""
+
+from __future__ import division, print_function
+
+import sys
+import os
+import shutil
+import abipy.abilab as abilab
+import abipy.data as abidata
+
+
+def help(stream=sys.stdout):
+    """
+    Display the tutorial text.
+    """
+    stream.write(__doc__)
```

Comment:
self.__doc__ ??

__file__.__doc__ ?

both not I guess :-(


Michiel van Setten
UCL, NAPS
mjvansetten@gmail.com
michiel.vansetten@uclouvain.be
010 478824

'"Although the way may not be obvious at first unless you're Dutch"
from The Zen of Python

On 21 Jan 2015, at 14:25, gmatteo wrote:

> In abipy/lessons/lesson_helper_functions.py:
> 
> > +"""
> > +
> > +from __future__ import division, print_function
> > +
> > +import sys
> > +import os
> > +import shutil
> > +import abipy.abilab as abilab
> > +import abipy.data as abidata
> > +
> > +
> > +def help(stream=sys.stdout):
> > +    """
> > +    Display the tutorial text.
> > +    """
> > +    stream.write(__doc__)
> this won't work because it returns the doc string of lesson_helper_functions.py
> 
> —
> Reply to this email directly or view it on GitHub.
>